addons/payment_stripe/models/payment_transaction.py

```diff
@@ -190,7 +191,7 @@ def _stripe_create_customer(self):
                 'address[postal_code]': self.partner_zip or None,
                 'address[state]': self.partner_state_id.name or None,
                 'description': f'Odoo Partner: {self.partner_id.name} (id: {self.partner_id.id})',
-                'email': self.partner_email or None,
```

Comment:
Fix directly in `payment`'s `payment.transaction::create` method so that it also works for all providers that correctly pass `tx.partner_email` instead of `tx.partner_id.email `.

Reply:
Done